build.gradle

```
@@ -53,10 +53,8 @@ subprojects { project ->
   }
 }
 
-configure(subprojects
-        - project("docs")
-) {
-  apply from: "https://raw.githubusercontent.com/arrow-kt/arrow/master/gradle/publication.gradle"
+configure(subprojects - project("docs")) {
+    apply from: file("${rootProject.rootDir}/gradle/publication.gradle")
```

```suggestion
configure(subprojects - project("docs")) {
    apply from: "https://raw.githubusercontent.com/arrow-kt/arrow/master/arrow-libs/gradle/publication.gradle"
```

Thank you so much @koperagen !! :raised_hands: So sorry, it has been relocated. This suggestion can replace the new local file.

Publish configuration is going to change because of the recent news about Bintray, Arrow and Arrow Meta will be released together very soon, and it's useful to keep just one configuration file.

Thank you again!! :clap:

---

Didn't noticed that this file has been relocated. I've deleted previously added local file and just replaced the URL as you suggest. Thank you ^^